lib/api/objectPutCopyPart.js

```diff
@@ -88,6 +98,7 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
         objectKey: destObjectKey,
         partNumber: paddedPartNumber,
         uploadId,
+        enableQuota: true,
```

Comment:
why do we need a second quota check in the `uploadPartCopy` ?

Reply:
Upload part copy acts as an upload part, but the part is an existing object: we need to evaluate the quotas in the same way as copyObject during the `get` part, because we write a new object...
Or maybe I misunderstood your question?

Reply:
I added a missing filter on the copy part API